### PR TITLE
Fixes #10 Give option to use YML file

### LINKED ISSUE #10: Give option to read yml file

### PR DESCRIPTION
**Pull request by RogerRussel**
With this update will be possible use a conf file, which can be Yaml or PHP file.

### CODE REVIEW

File: doc/conf/YAML.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Yaml - Configuration file
+
+To use read from Yaml file option it must has be installed.
```
Comment (alanyoshida) [inline doc/conf/YAML.md]: To use the Yaml conf file option it has to be installed.
To use the Yaml conf file option it must to be installed.

File: doc/conf/YAML.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+It it was not and, for some reason you don't want or don't can install it, you still can make it at regular php file, see more [here](./PHP.md).
```
Comment (alanyoshida) [inline doc/conf/YAML.md]: **If** it was not installed or for some other reason you don't want it or can't install it, you still can make it at regular php configuration file, see more [here](./PHP.md).

File: doc/conf/YAML.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Dependencies
+
+ * libyaml-dev
+
+### How install libyaml-dev
+
+ It work into my docker container, which was with debian image.
```
Comment (alanyoshida) [inline doc/conf/YAML.md]: It work into **the** docker container, which **is** an debian image.

File: doc/conf/YAML.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+ >`# apt-get install libyaml-dev -y`
+ >`# pecl install yaml`
+
+ Put this line bellow into php.ini file.
+
+ `extension=yaml.so`
+
+ I supose where you are using PHP will need be restarted.
```
Comment (alanyoshida) [inline doc/conf/YAML.md]: I **suppose** you will need to restart your PHP server.

File: doc/conf/YAML.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+```
+eyes:
+  MyEyeName:
+    type: Url
+    uri: http://UrlWhichIwantKeepOnEye
+    http:
+      method: get
+```
+
+2. Set to use the path to it
```
Comment (alanyoshida) [inline doc/conf/YAML.md]: Set to use the path to the yml file

File: doc/conf/PHP.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+# Php - Configuration file
+
+It's possible setup a conf file, which can be a PHP file or an Yml file, you can see more about Yml conf file at [here](YAML.md).
+
+## Example
+
+1. Write your conf file.
```
Comment (RogerRussel) [inline doc/conf/PHP.md]: Bellow not Above.

**Pull request merged**